terraform/dsd/iam/github_oidc.tf

```diff
@@ -20,7 +20,7 @@ data "aws_iam_policy_document" "github_actions_assume_role_policy_document" {
     condition {
       test     = "StringLike"
       variable = "${local.oidc_provider}:sub"
-      values   = ["repo:ministryofjustice/operations-engineering:*"]
+      values   = ["repo:ministryofjustice/operations-engineering:*", "repo:ministryofjustice/operations-engineering-dns-spike:*"]
```

Comment:
Is the role going to be used in both repos?

Reply:
It's to enable me to assume the role from GitHub Actions in another repository. This file is the github_oidc stuff